src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md

```diff
@@ -62,12 +58,6 @@ During [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud
 
 The following sections provide more details about CockroachDB {{ site.data.products.dedicated }} on Azure.
 
-### What is Limited Access and how does it relate to CockroachDB {{ site.data.products.dedicated }} on Azure?
-
-CockroachDB {{ site.data.products.dedicated }} on Azure is fully managed, like on GCP or AWS. During [limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cockroachdb-feature-availability), your {{ site.data.products.db }} organization must be enrolled before you can deploy CockroachDB {{ site.data.products.dedicated }} clusters on Azure.
-
-The clusters created during this period are recommended for proof-of-concept and testing, and are not suitable for production. The [CockroachDB {{ site.data.products.cloud }} Service Level Agreement (SLA)](https://cockroachlabs.com/cloud-terms-and-conditions/cockroachcloud-technical-service-level-agreement/) is not applicable to Azure clusters during limited access. Azure clusters are excluded from premium support agreements during limited access, and technical support is only available during business hours.
-
 ### Are multi-region clusters supported?
```

Comment:
Suggest adding that Serverless clusters are not supported

Reply:
Good idea, done.